crates/document_parse/Cargo.toml

```diff
@@ -10,7 +10,7 @@ version.workspace = true
 
 [dependencies]
 docx-rs = { git = "https://github.com/spiceai/docx-rs", rev="2d86a60b58afb02b157f96b42a92626417e47f71"}
-pdf-extract = { git = "https://github.com/spiceai/pdf-extract", rev = "3ca5e2c904cdf5897f3d671e3b2c4b62b1612b0c" }
+pdf-extract = { git = "https://github.com/spiceai/pdf-extract.git", rev = "d9a718b63b6b76b43086bb2c9976385a8e83762e" }
```

Comment:
Let's create a PR from your branch in `kczimm/lopdf-owner-encrypted-files` to the `spiceai` branch: https://github.com/spiceai/pdf-extract/tree/spiceai

This makes it easier to understand what code in our forks is actually being used in Spice. Once that PR to `spiceai/pdf-extract` is merged into the `spiceai` branch, you can update the commit hash here.

Reply:
Similarly for `https://github.com/spiceai/lopdf`, let's create a PR from `kczimm/fix-owner-encrypted-files` into the `spiceai` branch and use that commit in the `spiceai/pdf-extract` repo.

Reply:
You should also create an issue tracking the upstream work needed so we can eventually remove our forks. (i.e. see https://github.com/spiceai/spiceai/issues/4978)